pkg/backup/backup_new_test.go

```diff
@@ -1795,12 +1870,14 @@ func newHarness(t *testing.T) *harness {
 	}
 }
 
-func withLabel(obj metav1.Object, key, val string) metav1.Object {
+func withLabel(obj metav1.Object, labelPairs ...string) metav1.Object {
 	labels := obj.GetLabels()
 	if labels == nil {
 		labels = make(map[string]string)
 	}
-	labels[key] = val
+	for i := 0; i < len(labelPairs); i += 2 {
+		labels[labelPairs[i]] = labelPairs[i+1]
```

Comment:
@skriss this will panic if `labelPairs` is not even. I was thinking about catching that, but since the `testing.T` object isn't passed in here, we can't fail the test more gracefully. Not sure if we want to pass `t` into each `withLabel` call. We could also just panic with a more helpful error (e.g. "withLabel requires key-value pairs when specifying labels")

Reply:
I think panicking with a helpful error if there's an odd number of values should be fine. It's really just to detect programmer error.